python/cugraph/cugraph/link_analysis/pagerank.py

```diff
@@ -89,24 +103,41 @@ def pagerank(
     PageRank : cudf.DataFrame
         GPU data frame containing two cudf.Series of size V: the vertex
         identifiers and the corresponding PageRank values.
-
         df['vertex'] : cudf.Series
             Contains the vertex identifiers
         df['pagerank'] : cudf.Series
             Contains the PageRank score
-
-
     Examples
     --------
     >>> gdf = cudf.read_csv(datasets_path / 'karate.csv', delimiter=' ',
     ...                     dtype=['int32', 'int32', 'float32'], header=None)
     >>> G = cugraph.Graph()
-    >>> G.from_cudf_edgelist(gdf, source='0', destination='1')
+    >>> G.from_cudf_edgelist(gdf, source='0', destination='1', edge_attr='2')
     >>> pr = cugraph.pagerank(G, alpha = 0.85, max_iter = 500, tol = 1.0e-05)
-
     """
 
-    G, isNx = ensure_cugraph_obj_for_nx(G, weight)
+    initial_guess_vertices = None
+    initial_guess_values = None
+    pre_vtx_o_wgt_vertices = None
+    pre_vtx_o_wgt_sums = None
+
+    G, isNx = ensure_cugraph_obj_for_nx(G)
```

Comment:
It already has a default value [weight](https://github.com/rapidsai/cugraph/blob/branch-22.08/python/cugraph/cugraph/utilities/utils.py#L326) passed in and if `G` is of type `cugraph.Graph`, then the return is (`G`, `False`), otherwise, it will extract the [edges](https://github.com/rapidsai/cugraph/blob/branch-22.08/python/cugraph/cugraph/utilities/nx_factory.py#L44)(`src, dst, weight`)  from the `nx.Graph` and return a [cudf.DataFrame](https://github.com/rapidsai/cugraph/blob/branch-22.08/python/cugraph/cugraph/utilities/nx_factory.py#L55) which will be used to create a `cugraph.Graph`

Reply:
Resolved by reverting this line, so it's now passing `weight` as before.